src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -304,7 +304,7 @@ export const WorkspaceDashboard = _.flow(
               }, [icon('times', { size: 14 })])
             ])
           }, tagsList),
-          busy && spinner({ style: { marginTop: '0.5rem' } })
+          (busy || tagsList === undefined) && spinner({ style: { marginTop: '0.5rem' } })
```

Comment:
This seems to create more movement when a workspace has no tags, moving the "Google Bucket..." section down to fit the spinner and then back up when the page has finished rendering. Maybe could add a height to the container so that the space is consistently large enough to accommodate the spinner?